src/solvers/rosenbrock.jl

```diff
@@ -1,102 +1,589 @@
-export SSPKnoth
+#=
+## Introduction to Rosenbrock Methods
 
-abstract type RosenbrockAlgorithm <: DistributedODEAlgorithm end
+An s-stage diagonally implicit Runge-Kutta (DIRK) method for solving
+∂u/∂t = f(u, t) is specified by a lower triangular s×s matrix a (whose values
+are called "internal coefficients"), a vector b of length s (whose values are
+called "weights"), and a vector c of length s (whose values are called
+"abcissae" or "nodes").
+Given the state u at time t, the state u_next at time t + Δt is given by
+    u_next := u + Δt * ∑_{i=1}^s bᵢ * f(Uᵢ, t + Δt * cᵢ), where
+    Uᵢ := u + Δt * ∑_{j=1}^i aᵢⱼ * f(Uⱼ, t + Δt * cⱼ).
+In order to transform this DIRK method into a Rosenbrock method, we must assume
+that it is "internally consistent", which means that
+    cᵢ := ∑_{j=1}^i aᵢⱼ.
 
-struct RosenbrockTableau{N, RT, N²}
```

Comment:
Wait, we already had a Rosenbrock time stepper? How is what was added different? Can this please be elaborated in the description? The previous implementation (that it looks like @simonbyrne implemented?) looks much simpler

Reply:
I have a similar question, perhaps phrased slightly differently: Why does enabling limiters necessitate this much more complexity than the straightforward Rosenbrock implementation in the older code?